src/PSDMatrices.jl

```diff
@@ -37,9 +37,10 @@ function det(M::PSDMatrix)
     return det(M.R)^2
 end
 
-function logdet(M::PSDMatrix)
+function logabsdet(M::PSDMatrix)
     confirm_factor_is_square(M)
-    return 2 * logdet(M.R)
+    _logabsdet, _sign = logabsdet(M.R)
+    return 2 * _logabsdet, _sign^2
```

Comment:
why the underscores? And is there any case where `_sign^2` is not equal to 1?

Reply:
I didn't want to overwrite the logabsdet function, so I just added an underscore. And then did the same to sign for coherence. If you'd prefer a different way I can change it?

And yes, _sign^2 is always one I guess, which makes sense since we're doing PSDMatrices. I'll change that :+1:

Reply:
With the latest change I also removed the underscored.